### PR TITLE
feat: purge default storages even without `Actor.init`

### PR DESCRIPTION
**Pull request by B4nan**

### CODE REVIEW

**Review by AndreyBykov — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 * (internal) `Request.handledAt` is now string date in ISO format
 * (perf/internal) `Request.inProgress` and `Request.reclaimed` are now `Set`s instead of dictionaries
 * `injectUnderscore` from puppeteer utils has been removed
-* `APIFY_MEMORY_MBYTES` is no longer taken into account, use `AVAILABLE_MEMORY_RATIO` instead
+* `APIFY_MEMORY_MBYTES` is no longer taken into account, use `CRAWLEE_AVAILABLE_MEMORY_RATIO` instead
```
Comment (mnmkng) [inline CHANGELOG.md:23]: Ratio? So it's no longer mbytes but percentage of system memory?

File: packages/apify/src/actor.ts [hunk 2]
```diff
@@ -399,7 +396,7 @@ export class Actor<Data extends Dictionary = Dictionary> {
             customAfterSleepMillis = this.config.get('metamorphAfterSleepMillis'),
             ...metamorphOpts
         } = options;
-        const runId = this.config.get('actorRunId')!;
+        const runId = process.env[ENV_VARS.ACTOR_RUN_ID]!;
```
Comment (mnmkng) [inline packages/apify/src/actor.ts:399]: Why are we coming back to reading process envs? I thought the configuration would read the envs and then everything else would read it from configuration.

File: CHANGELOG.md (hunk 1 above)
Comment (B4nan) [reply CHANGELOG.md:23]: Yes, we use 1/4 of available memory by default, and this limit is removed when using `Actor.init` and we are on platform, its based on the [previous implementation](https://github.com/apify/apify-js/blob/5d372cbeade2ec78658d84ea6a3e9893598229b5/src/autoscaling/snapshotter.js#L479). 

But I do plan to support exact setting of memory limit too, to keep the

File: packages/apify/src/actor.ts (hunk 2 above)
Comment (B4nan) [reply packages/apify/src/actor.ts]: Removed those changes for now, I guess we should have that second config class in actor sdk if we want to support those too.

**Pull request merged**